v1/InhibitRules.dhall

```diff
@@ -0,0 +1,18 @@
+let imports = ../imports.dhall
+
+let Matcher = ./Matcher.dhall
+
+let InhibitRules =
```

Comment:
Should change to `InhibitRule` instead of `InhibitRules` including the file name